### PR TITLE
Configure Rubocop with ruboconfig

### PR DESCRIPTION
**Pull request by timrogers**

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Configuration parameters: ExpectMatchingDefinition, Regex, IgnoreExecutableScripts, AllowedAcronyms.
+# AllowedAcronyms: CLI, DSL, ACL, API, ASCII, CPU, CSS, DNS, EOF, GUID, HTML, HTTP, HTTPS, ID, IP, JSON, LHS, QPS, RAM, RHS, RPC, SLA, SMTP, SQL, SSH, TCP, TLS, TTL, UDP, UI, UID, UUID, URI, URL, UTF8, VM, XML, XMPP, XSRF, XSS
+Naming/FileName:
+  Exclude:
+    - 'lib/activerecord-safer_migrations.rb'
```
Comment (timrogers) [inline .rubocop_todo.yml]: I couldn't find any way to rename this to keep the cop happy *and* keep existing behaviour.

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+# Offense count: 20
+# Configuration parameters: AllowedVariables.
+Style/GlobalVars:
+  Exclude:
+    - 'spec/active_record/safer_migrations/migration_spec.rb'
```
Comment (timrogers) [inline .rubocop_todo.yml]: The structure of these tests makes the global variable hard to kill.

**Comment by timrogers** [comment]
@nickcampbell18 Could you give this very old PR a review? I've just updated it to the latest and greatest.

File: lib/active_record/safer_migrations/postgresql_adapter.rb [hunk 3]
```diff
@@ -3,23 +3,25 @@
...
-      SET_SETTING_SQL = <<-SQL.freeze
+      SET_SETTING_SQL = <<-SQL.
       UPDATE
         pg_settings
       SET
         setting = :value
       WHERE
         name = :setting_name
       SQL
+        freeze
```
Comment (nickcampbell18) [inline lib/active_record/safer_migrations/postgresql_adapter.rb]: This is really messed up.

Comment (timrogers) [reply lib/active_record/safer_migrations/postgresql_adapter.rb]: Rubby, ladies and gentlemen

Comment (Sinjo) [reply lib/active_record/safer_migrations/postgresql_adapter.rb]: rubby_way

**Pull request merged**